### DISCUSSION TITLE
New Release - Happy Holidays

### DISCUSSION THREAD

**Discussion by vladmandic** [vladmandic/sdnext#2654] [Announcements]
## Happy holidays everyone!

To wrap up this amazing year, we're releasing a new version of [SD.Next](https://github.com/vladmandic/automatic), this one is absolutely massive!  
For full list of changes, see [Changelog](https://github.com/vladmandic/automatic/blob/master/CHANGELOG.md) *(it really is too long to post here)*  

### Highlights  

- Brand new Control module for *text, image, batch and video* processing  
  Native implementation of all control methods for both *SD15* and *SD-XL*  
  ▹ **ControlNet | ControlNet XS | Control LLLite | T2I Adapters | IP Adapters**  
  For details, see [Wiki](https://github.com/vladmandic/automatic/wiki/Control) documentation:  
- Support for new models types out-of-the-box  
  This brings number of supported t2i/i2i model families to 13!  
  ▹ **Stable Diffusion 1.5/2.1 | SD-XL | LCM | Segmind | Kandinsky | Pixart-α | Würstchen | aMUSEd | DeepFloyd IF | UniDiffusion | SD-Distilled | BLiP Diffusion | etc.**  
- New video capabilities:  
  ▹ **AnimateDiff | SVD | ModelScope | ZeroScope**  
- Enhanced platform support  
  ▹ **Windows | Linux | MacOS** with **nVidia | AMD | IntelArc | DirectML | OpenVINO | ONNX+Olive** backends  
- Better onboarding experience (first install)  
  with all model types available for single click download & load (networks -> reference)  
- Performance optimizations!
  For comparisment of different processing options and compile backends, see [Wiki](https://github.com/vladmandic/automatic/wiki/Benchmark)  
  As a highlight, we're reaching **~100 it/s** (no tricks, this is with full features enabled and end-to-end on a standard nVidia RTX4090)  
- New [custom pipelines](https://github.com/vladmandic/automatic/blob/dev/scripts/example.py) framework for quickly porting any new pipeline  

And others improvements in areas such as: Upscaling (up to 8x now with 40+ available upscalers), Inpainting (better quality), Prompt scheduling, new Sampler options, new LoRA types, additional UI themes, better HDR processing, built-in Video interpolation, parallel Batch processing, etc.  

Plus some nifty new modules such as **FaceID** automatic face guidance using embeds during generation and **Depth 3D** image to 3D scene

Links: [Home](https://github.com/vladmandic/automatic) | [Readme](https://github.com/vladmandic/automatic/blob/master/README.md) | [Changelog](https://github.com/vladmandic/automatic/blob/master/CHANGELOG.md) | [Wiki](https://github.com/vladmandic/automatic/wiki) | [Discord](https://discord.gg/VjvR2tabEX)

![](https://vladmandic.github.io/automatic/html/xmas-default.jpg)  
![](https://vladmandic.github.io/automatic/html/xmas-control.jpg)

**Comment by Sunsphire** [comment]
Happy Holidays to you too!!! Thank you a lot for all the work you and the team did, words are not enough to express my gratitude.

Since this is a massive update do you recommend doing a fresh install or a normal update is enough?

**Reply by vladmandic** [reply]
Reinstall is never a bad idea for a major release, but it's not actually required this time.

**Comment by gohan2091** [comment]
Awesome, thanks for your time and effort :)

**Comment by Morbleh** [comment]
how to make a theme like the one on the second screen?

**Reply by vladmandic** [reply]
theme in second screen is same as in first. interface is new because that is what new release is about.
